_posts/2018-02-06-bazel-0.10.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: posts
+title: "Bazel 0.10"
+---
+
+We're proud to announce the release of [Bazel 0.10](https://github.com/bazelbuild/bazel/releases/tag/0.10.0).
+The 400+ commits since last release include performance optimizations, bug fixes,
+and various improvements.
+
+There is a new android test rule. `android_local_test` tests `android_library`
+code using Robolectric, a unit test framework designed for test-driven
+development without the need for an emulator or device.
+
+The depset type has evolved. To merge multiple depsets or add new elements, use
+instead the new depset constructor, which has a better performance. The
```

Comment:
First, say what people should use the ctor instead of, before saying that they should use it instead of something. That's easier to read than the current phrasing.

E.g.:
+The depset type has evolved. To merge multiple depsets or add new elements, do not
+use the operators (...), use the new depset constructor instead, which has better performance.
+We will deprecate and remove the operators in the future.

Reply:
Done.